README.md

```diff
@@ -161,9 +163,9 @@ may differ between network types (e.g., estimations often based on frequencies
 of interactions in plant-pollinator networks), the broad principles underlying
 our findings remain relevant and applicable across diverse ecological contexts.
 
-# Probabilistic representations of interactions
+# Representations of probabilistic interaction webs
```

Comment:
L166: My idea here was to have matching section titles. The one for the next section is "Properties of probabilistic interaction webs"